### DISCUSSION TITLE
Trapped at final part of application

### DISCUSSION THREAD

**Discussion by ixnoahlive** [community/community#69191] [GitHub Education]
### Hi everyone,

I've been trying to apply to GitHub education for a while, but every time I do it just refreshes on the final step after processing.
I was rejected twice previously but I don't know if that has anything to do with it.

**Comment by samus-aran** [comment]
Hi @NoahTheNerd 

Sorry to hear that. Is this when you are uploading your evidence? Are you able to open your browser's developer tools and see if there are any console logs/errors being reported during this process.

**Reply by ixnoahlive** [reply]
Will look into the console when I have the time. Cheers.

**Reply by ixnoahlive** [reply]
![image](https://github.com/community/community/assets/70314622/3d5a7bf5-270f-4724-b662-6152138f4cbf)
![image](https://github.com/community/community/assets/70314622/f1bb9035-dcce-4fae-8edb-ad3944ab9bb7)
